frontend/src/components/Accounts/AccountDetailsAccountStatement.vue

```diff
@@ -133,13 +133,20 @@
 			<div class="col-span-1 flex justify-end">
 				<a
 					class="bg-theme-button-primary px-8 py-3 hover:bg-theme-button-primary-hover rounded"
-					:href="exportUrl"
+					:href="exportUrl(accountAddress, chosenMonth)"
 				>
 					<span class="hidden md:inline">Export</span>
 					<DownloadIcon class="h-4 inline align-text-top" />
 				</a>
 			</div>
 		</div>
+		<div class="col-span-1 flex justify-end">
+			<input
+				v-model="chosenMonth"
+				class="bg-theme-background-primary-elevated-nontrans rounded"
+				type="month"
```

Comment:
Would be good to set `min` (date of account creation or chain genesis if not feasible) and `max` (current date) properties to limit the range of months you can select.

Reply:
Added.